app/queries/scrape_result_search_query.rb

```diff
@@ -19,16 +19,27 @@ def initialize(pattern:, query_type:)
   end
 
   def call
+    search_entries_filtered_by_url
+      .group(:keyword_id)
+      .includes(:keyword)
```

Comment:
If I don't want to preload/include all fields from keyword, how can I achieve that 🤔 ?

For example, in this case, I only need the `content` field from the keyword table.

Reply:
You would need to use `.select`:

```suggestion
      .includes(:keyword)
      .select('keyword_search_entry.keyword_id, keyword_search_entry.urls, keyword.content')
```

However, unsure if it is worth it in this case to optimize the queries.

Reply:
Oh, I'm just curious how this be done